netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -797,101 +901,227 @@ _net_ebpf_extension_connection_context_initialize(
     }
 }
 
-static net_ebpf_extension_connection_context_t*
-_net_ebpf_ext_get_and_remove_connection_context(
+_Requires_exclusive_lock_held_(
+    _net_ebpf_ext_sock_addr_blocked_contexts
+        .lock) static bool _net_ebpf_ext_find_and_remove_connection_context_locked(_In_
+                                                                                       net_ebpf_extension_connection_context_t*
+                                                                                           context)
```

Comment:
Indentation looks weird here, is this really what the format code script did?

Reply:
Yeah, this is what the code formatted did. It looks like many of the function annotations (_requires*, _IRQL_requires*, etc.) are all a bit weird in this file.